extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -98,16 +100,24 @@ function render_player( $track_list, $attributes ) {
 	}
 
 	ob_start();
-
 	?>
-	<div class="<?php echo esc_attr( $block_classname ); ?>">
-		<?php // Placeholder: block markup is being handled in https://github.com/Automattic/jetpack/pull/14952. ?>
-		<script type="application/json" class="wp-playlist-script"><?php echo wp_json_encode( $player_data ); ?></script>
+	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
+		<ol>
+			<?php
+			foreach ( $track_list as $att_id => $attachment ) :
+				printf( '<li><a href="%1$s" data-jetpack-podcast-audio="%2$s">%3$s</a></li>', esc_url( $attachment['link'] ), esc_url( $attachment['src'] ), esc_html( $attachment['title'] ) );
```

Comment:
Although I love the progressive enhancement here of using a link that gets enhanced into a playlist, I think this should be a button. It should be able to be activated by a spacebar press.

So, should we:
1. use things like `role="button"` and add a keypress modifier
2. use a `<button>` by default 
3. use this current output as a fallback then have the player output its own playlist using `<button>`s? 

3 is probably the best, although most work.

Reply:
I plan to rework the function for playing a song in a separate PR - would you mind if I did the change there? I think about 1 as the go-to variant since, 2 would render the semantic markup unusable and 3 is, as you mention, a lot of work we have not really planned for.

Reply:
I'm working on it in a separate PR already: https://github.com/Automattic/jetpack/pull/14989/files#diff-551fe0712dea509839f9d2b144489d57

The list markup and styling are almost done, but they definitely need an aria look. Can we move that discussion to https://github.com/Automattic/jetpack/pull/14989, please?